pkg/neorpc/result/peers.go

```diff
@@ -63,3 +64,38 @@ func (p *Peers) addPeers(addrs []string) {
 		*p = append(*p, peer)
 	}
 }
+
+func (p *Peer) UnmarshalJSON(data []byte) error {
+	type NewPeer Peer
```

Comment:
Do we need this type?

Reply:
Yes or we get infinite recursion on the `json.Unmarshal` call